easybuild/tools/package/utilities.py

```diff
@@ -173,20 +193,20 @@ def __init__(self):
         if sel_pns in avail_pns:
             self.pns = avail_pns[sel_pns]()
         else:
-            raise EasyBuildError("Selected package naming scheme %s could not be found in %s",
-                                 sel_pns, avail_pns.keys())
+            raise EasyBuildError("Selected package naming scheme %s could not "
+                                 "be found in %s", sel_pns, avail_pns.keys())
 
-    def name(self, ec):
+    def name(self, easyconfig):
         """Determine package name"""
-        name = self.pns.name(ec)
+        name = self.pns.name(easyconfig)
         return name
 
-    def version(self, ec):
+    def version(self, easyconfig):
         """Determine package version"""
-        version = self.pns.version(ec)
+        version = self.pns.version(easyconfig)
         return version
 
-    def release(self, ec):
+    def release(self, easyconfig):  # pylint: disable=W0613
```

Comment:
shouldn't it be passed down to `self.pns.release` (which can then choose to ignore it)?